t.py

```diff
@@ -12,7 +12,7 @@
 def setData(user):
     try:
         user['username'] = raw_input("Enter Your Account:")
-    except:
+    except NameError:
```

Comment:
Use sys.version to check python version: [like this](https://github.com/moskytw/mosql/blob/4b2c58c03ed5f0be2a97cea922319871369df1df/mosql/compat.py#L33)

Reply:
Also, this is not a coding style fix